packages/contracts/contracts/interpreters/FulfillInterpreter.sol

```diff
@@ -73,7 +73,8 @@ contract FulfillInterpreter is ReentrancyGuard, IFulfillInterpreter {
     // Check if the callTo is a contract
     bool success;
     bytes memory returnData;
-    if (Address.isContract(callTo)) {
+    bool isContract = Address.isContract(callTo);
+    if (isContract) {
```

Comment:
so we don't execute the call data if it's not targeting a contract then? who executes the call data then, exactly? sorry i'm fairly certain this is a really basic q